apps/mobile/src/components/Feed/Socialize/CommentBox.tsx

```diff
@@ -92,45 +79,43 @@ export function CommentBox({
   }, [showXMark, width, display]);
 
   return (
-    <View className="px-2 pb-2 flex flex-row items-center space-x-3">
-      <Animated.View className="flex-1 flex-row justify-between items-center bg-faint dark:bg-black-800 p-1.5 space-x-3">
-        <BottomSheetTextInput
-          value={value}
-          onChangeText={setValue}
-          className="text-sm h-5"
-          selectionColor={colorScheme === 'dark' ? colors.white : colors.black['800']}
-          autoCapitalize="none"
-          autoComplete="off"
-          autoFocus={autoFocus}
-          onBlur={handleDismiss}
-          onSubmitEditing={handleDismiss}
-          keyboardAppearance={colorScheme}
-          style={{ flex: 1, color: colorScheme === 'dark' ? colors.white : colors.black['800'] }}
-        />
-        <Text className="text-sm text-metal">{characterCount}</Text>
-        <GalleryTouchableOpacity
-          eventElementId="Submit Comment Button"
-          eventName="Submit Comment Button Clicked"
-          onPress={handleSubmit}
-          disabled={disabledSendButton}
-        >
-          <View
-            className={`h-6 w-6 rounded-full flex items-center justify-center bg-red
+    <View
+      className={`border-t-2 border-solid ${
+        colorScheme === 'dark' ? `border-[${colors.black[500]}]` : 'border-[#e2e2e2]'
+      }`}
+    >
+      <View className="px-2 pb-2 flex flex-row items-center space-x-3 mt-2">
```

Comment:
the new `mt-2` is messing up the positioning of the input , you'll see in your screenshot that it's pushing the input down so that the spacing between it and keyboard is no longer there.
![CleanShot 2023-08-14 at 19 34 02](https://github.com/gallery-so/gallery/assets/80802871/01c3a25c-f6e5-4c76-b007-a70e145a7fbc)


the visible bottom sheet area above the keyboard is a fixed height so when we make it's contents taller, things get pushed below the visible area - we experienced this with your [other PR](https://github.com/gallery-so/gallery/pull/1753#issuecomment-1665580119) 

looks like you'll need to account for this by taking a look at this handle component that's currently contributing the top spacing https://github.com/gallery-so/gallery/blob/25df86246c78deacbcd55ec3315983093940c80a/apps/mobile/src/components/Feed/Socialize/CommentButton.tsx#L93C1-L93C1